main.c

```diff
@@ -47,12 +48,12 @@
 #define FIELD_X GAME_W - BULLET_S * 2
 #define STEP_Y 6
 
-#define WINDOW_W GAME_W
-#define WINDOW_H GAME_H
+#define WINDOW_W 640
+#define WINDOW_H 480
 #if __EMSCRIPTEN__
 #define RECORDS_FILE "IDBFS/records.bin"
 #else
-#define RECORDS_FILE "records.bin"
+//#define RECORDS_FILE "records.bin"
```

Comment:
Use the UDATA folder on "E:"; otherwise the game might crash as the current folder of the binary might be on a read-only DVD.